hw/ip/pinmux/data/pinmux.hjson.tpl

```diff
@@ -410,14 +665,15 @@
                 }
 
     },
-# wakeup detector config
-    { multireg: { name:     "WKUP_DETECTOR",
-                  desc:     "Configuration of wakeup condition detectors."
-                  count:    "NWkupDetect",
-                  swaccess: "rw",
-                  hwaccess: "hro",
-                  regwen:   "REGEN",
-                  cname:    "DETECTOR",
+    { multireg: { name:         "WKUP_DETECTOR",
+                  desc:         "Configuration of wakeup condition detectors."
+                  count:        "NWkupDetect",
+                  compact:      "false",
+                  swaccess:     "rw",
+                  hwaccess:     "hro",
+                  regwen:       "WKUP_DETECTOR_REGWEN",
+                  regwen_multi: "true",
+                  cname:        "DETECTOR",
```

Comment:
is there a need for a disabled mode if there's also a `wkup_detector_en`?

Reply:
no this seems redundant. let me remove that.